### PR TITLE
Feafure/navigation

### PR DESCRIPTION
**Pull request by KirillSerg**
- implement UI of left navigation bar
- configure all navigation routes using react-router-dom
- add empty pages for each route
- - connect icons library Bootstrap
image

### CODE REVIEW

File: src/components/NavigationBar.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { Nav } from "react-bootstrap";
```
Comment (KirillSerg) [inline src/components/NavigationBar.tsx:1]: oh crap((
I will fix it

File: src/components/NavigationBar.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  Hash,
+  Envelope,
+} from "react-bootstrap-icons";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const StyledNav = styled(Nav)`
+  background-color: white;
+  color: black;
+  .nav-link,
```
Comment (Tretmax) [inline src/components/NavigationBar.tsx:18]: maybe you need to add an indent. Text too close to icon

Comment (DavidKorystov) [reply src/components/NavigationBar.tsx:18]: You also should align the list items vertically

File: src/components/LeftPanel.tsx [hunk 3]
```diff
@@ -28,7 +31,10 @@ const Wrapper = styled.div`
 export const LeftPanel = () => {
   return (
     <Wrapper>
-      <Div>left side</Div>
+      <Div>
+        <Twitter />
```
Comment (DavidKorystov) [inline src/components/LeftPanel.tsx]: You should add some space for the "bird" from above and the left, like on this photo

image

**Review by DavidKorystov — APPROVED** [review]
Nice work

**Comment by ArtemLigerko** [comment]
Mistake in name branch 'feafure/navigation' should be 'feature/navigation'

File: src/components/LeftPanel.tsx (hunk 3 above)
Comment (alexandrtovmach) [inline src/components/LeftPanel.tsx]:
```suggestion
        <Logo />
```

File: src/containers/Layout.tsx [hunk 4]
```diff
@@ -41,6 +41,13 @@ const Layout = ({ children, withoutNavigation }: LayoutProps) => {
             <li>
               <Link to="/">Home</Link>
               <Link to="/about">About</Link>
+              <Link to="/explore">Explore</Link>
+              <Link to="/notifications">Notifications</Link>
+              <Link to="/messages">Messages</Link>
+              <Link to="/bookmarks">Bookmarks</Link>
+              <Link to="/lists">Lists</Link>
+              <Link to="/profile">Profile</Link>
+              <Link to="/more">More</Link>
```
Comment (alexandrtovmach) [inline src/containers/Layout.tsx]: I think it's not required to add these links if you already added navigation links in NavigationBar

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
few changes and will be perfect

**Review by Mykyta-Pysarev — APPROVED** [review]
Add paddings between icons and links. Overall seems ok.

File: src/containers/Layout.tsx (hunk 4 above)
Comment (KirillSerg) [reply src/containers/Layout.tsx]: Yeah, that's what I thought too, but I wasn't sure. Can generally remove the old test navigation from Layou?

**Review by alexandrtovmach — APPROVED** [review]
pide

**Review by AndrewKaenko — APPROVED** [review]

**Pull request merged**